lib/src/types/did_url.rs

```diff
@@ -54,7 +64,37 @@ impl ToString for Account {
     fn to_string(&self) -> String {
         match self {
             Account::Address(addr) => format!("0x{}", hex::encode(addr.as_bytes())),
-            Account::HexKey(key) => format!("0x{}", hex::encode(key)),
+            Account::HexKey(key) => hex::encode(key).to_string(),
+        }
+    }
+}
+
+impl Account {
+    pub fn from_hex_key(key: Vec<u8>) -> Self {
```

Comment:
maybe use From would be more idiomatic?
```rust
impl From<HexKey/Address/&str> for Account {
    // implementation
}
```

Reply:
💯💯 thank you this feedback is so helpful, I did this method kind of fast and forgot to even think about From 🙈